app/src/test/java/org/simple/clinic/newentry/PatientEntryScreenLogicTest.kt

```diff
@@ -166,6 +167,42 @@ class PatientEntryScreenLogicTest {
     verify(ui).openMedicalHistoryEntryScreen()
   }
 
+  @Test
+  fun `when bangladesh national id is entered then patient record should still be created`() {
```

Comment:
Can we rename this test to be more specific?
`when bangladesh national id is entered and save is clicked then patient should get registered`

Reply:
Done